### ISSUE TITLE
Cinnamon Settings 

### ISSUE THREAD

**Issue by robin0800** [linuxmint/cinnamon#10933]
```
 * Cinnamon version 5.42
 * Distribution - Manjaro 21.3.3
 * Graphics hardware NVIDIA GeForce GT 1030 NVIDIA Driver Version: 515.57
 * 64 bit
 
 ```

**Issue**
cinnamon-settings won't open 

**Steps to reproduce**

Try to open cinnamon-settings

**Expected behaviour**
cinnamon-settings opens

**Other information**
/usr/share/cinnamon/cinnamon-settings/modules/secs_notifications.py needs an update as notify has been updated to version 0.8.
Error with 0.7 in the file

```
[robin0800@robins-desktop ~]$ cinnamon-settings
Traceback (most recent call last):
  File "/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py", line 796, in <module>
    window = MainWindow()
  File "/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py", line 320, in __init__
    self.init_settings_overview()
  File "/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py", line 328, in init_settings_overview
    self.load_python_modules()
  File "/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py", line 497, in load_python_modules
    for module in map(__import__, to_import):
  File "/usr/share/cinnamon/cinnamon-settings/modules/cs_notifications.py", line 4, in <module>
    gi.require_version('Notify', '0.7')
  File "/usr/lib/python3.10/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available for version %s' %
ValueError: Namespace Notify not available for version 0.7
```

**Comment by Fantu** [comment]
https://gitlab.gnome.org/GNOME/libnotify/-/issues/27
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1015006

**Comment by robin0800** [comment]
Yes all fixed now and working with 0.7 again

**Comment by Fantu** [comment]
don't result fixed upstream, probably your distribution added a temp. workaround (and probably I'll add also in cinnamon for debian experimental and unstable)
